### ISSUE TITLE
[Java-Playframework] UUID parameter is not well parsed

### ISSUE THREAD

**Issue by tzimisce012** [swagger-api/swagger-codegen#6381]
##### Description

If a query parameter or header has the format UUID, this is not well parsed and the code generated don't compile

##### Swagger-codegen version

2.3.0-SNAPSHOT

##### Swagger declaration file content or url

For a swagger snipet like:

```
          "parameters": {
              "Correlator": {
                "type": "string",
                "format": "uuid",
                "description": "Correlation id for the different services"
              }
            },
```

the code generate is this:

```
String valuecorrelator = request().getHeader("Correlator");
        UUID correlator;
        if (valuecorrelator != null) {
            correlator = (String)valuecorrelator;

```

and this will not compile.

##### Command line used for generation

language JavaPlayFramework

##### Suggest a fix/enhancement

Add the option to parse UUID with `UUID.fromString()` method

**Comment by JFCote** [comment]
This will have to be checked by @wing328 and other people who know the project better than me.

The problem I see is this (from the documentation)

> Primitives have an optional modifier property format. Swagger uses several known formats to more finely define the data type being used. However, the format property is an open string-valued property, and can have any value to support documentation needs. Formats such as "email", "uuid", etc., can be used even though they are not defined by this specification. Types that are not accompanied by a format property follow their definition from the JSON Schema (except for file type which is defined above).

Even if the spec is considered "valid" when using formats such as "email" and "uuid", I understand the doc is considering them as "custom" and I'm not sure tools like swagger-codegen should implements all custom field formats in the world. But maybe it's ok and should be implemented. I will review your PR with my comments in case it should be implemented.

**Comment by wing328** [comment]
Closed via #6382
